app/javascript/application.js

```diff
@@ -27,6 +28,21 @@ $(document).ready(function() {
 
   var $body = $(document.body);
 
+  $(".clipboard-btn").on("click", function(){
+    let $button = $(this);
+    let $token = $(".solution-token");
+
+    let copySrc = $button.find('img').attr('src');
+    let successSrc = $button.attr('data-success');
+
+    Clipboard.copy($token.text());
+
+    $button.find('img').attr('src', successSrc);
+    setTimeout(() => {
+      $button.find('img').attr('src', copySrc);
+    }, 1000);
+  });
```

Comment:
The code picks up "this" which is the element we targeted with `.clipboard-btn`, the button. In the button, there's an `img` tag which has the "copy" icon's image URL. We also need a "success" icon, which I've attached as a `data-success` attribute. 

Once we copy the token's text, we switch the src attribute of the image to a ✅ and after 1 second, we switch it back.

Reply:
Maybe switch to 2000 and use image_tag resize to resize the copy.png because it is misaligned a little

Reply:
Images and emoji are difficult to mix, since emoji is text and is affected by it, and it varies across systems. To ensure consistency between the two buttons, I've changed the copy button to be a link as well and put 📋 as text instead of an image.